public/swagger/swagger.json

```diff
@@ -0,0 +1,997 @@
+            "schema": {
+              "$ref": "#/definitions/ApiResponse"
+            }
+          }
+        }
+      }
+    },
+    "/namespace/{namespaceId}/rollup/{ruleId}/history": {
+      "get": {
+        "tags": [
+          "rollup-rule"
+        ],
+        "summary": "Gets the current state and all history for a given rollup rule in a given  namespace\n",
+        "operationId": "getRollupRuleHistory",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-USER-ID",
+            "description": "the user making this request",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "namespaceId",
+            "description": "The name of the namespace",
+            "type": "string",
+            "required": true
+          },
+          {
+            "in": "path",
+            "name": "ruleId",
+            "description": "The id of the rule",
+            "type": "string",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "rule state sorted latest first",
+            "schema": {
+              "$ref": "#/definitions/RollupRuleHistory"
+            }
+          },
+          "404": {
+            "description": "no such namespace or no such rule",
+            "schema": {
+              "$ref": "#/definitions/ApiResponse"
+            }
+          },
+          "500": {
+            "description": "Something went horribly wrong",
+            "schema": {
+              "$ref": "#/definitions/ApiResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Namespace": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      }
+    },
+    "Namespaces": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Namespace"
+      }
+    },
+    "RuleSet": {
+      "type": "object",
+      "properties": {
+        "namespace": {
+          "type": "string"
+        },
+        "version": {
+          "type": "integer"
+        },
+        "cutoverTime": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "mappingRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MappingRule"
+          }
+        },
+        "rollupRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RollupRule"
+          }
+        }
+      }
+    },
+    "MappingRule": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "uuid"
+        },
+        "name": {
+          "type": "string"
+        },
+        "filters": {
```

Comment:
Also at some point, I would add what a valid "filter" and "policy" is in the top of the docs.